### PR TITLE
Do not load records from LSS which are marked as excluded

### PR DESCRIPTION
**Pull request by cnorthwood**
In earlier versions of the data, these would be pre-filtered out. Now we're provided with a flag in the CSV and have to check ourselves.

### CODE REVIEW

File: spec/lss_loader_spec.rb [hunk 1]
```diff
@@ -121,6 +121,12 @@
     expect_basic_hierarchy
   end
 
+  it "ignores advice locations which are marked as excluded" do
```
Comment (leeky) [inline spec/lss_loader_spec.rb:124]: Would it be useful to have individual tests for each of the three states that mark an advice location as 'excluded' ?

**Review by leeky — APPROVED** [review]
LGTM! :shipit:

File: spec/lss_loader_spec.rb (hunk 1 above)
Comment (cnorthwood) [reply spec/lss_loader_spec.rb:124]: I think the first one is covered by another test (checks for data integrity), but the latter 2 could be separate, but I guess there's a discussion about how many cases a test should cover, every combination etc

Comment (leeky) [reply spec/lss_loader_spec.rb:124]: Gotcha. I tend to agree about having a minimal amount of tests, but in the case where the logic is ensuring that a future change won't risk excluded locations from showing up on the public website, I tend to err on the side of caution.

**Pull request merged**